src/vm/disassembler.h

```diff
@@ -14,7 +14,7 @@
         // COREDISTOOLS disassembler only supports amd64, so if this is
```

Comment:
Shouldn't the comment be updated as well to indicate support for both amd64 and x86?